src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,25 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <UserInfo user={post.user} />
+      </div>
+
+      <p className="PostInfo__body">{post.body}</p>
+
+      <hr />
+
+      {post.comments.length > 0 ? (
+        <CommentList comments={post.comments} />
+      ) : (
+        <b data-cy="NoCommentsMessage">No comments yet</b>
```

Comment:
```suggestion
      {post.comments.length > 0 
        ? <CommentList comments={post.comments} />
        :  <b data-cy="NoCommentsMessage">No comments yet</b>
```

Reply:
Unfortunatly I can't change it because of linter settings